optaweb/tests/features/optaweb.feature

```diff
@@ -0,0 +1,12 @@
+@rhdm-7/rhdm70-optaweb
+Feature: RHDM Employee Rostering Optimizer tests
```

Comment:
Ideally, this could be the description minus suffix "container image" plus suffix "tests"

Reply:
change to: 
Feature: Red Hat Business Optimizer OptaWeb Employee Rostering 7.1 tests